ports/windows/variants/dev/mpconfigvariant.mk

```diff
@@ -0,0 +1,5 @@
+PROG ?= micropython-dev.exe
+
+FROZEN_MANIFEST ?= $(VARIANT_DIR)/manifest.py
+
+MICROPY_ROM_TEXT_COMPRESSION = 1
```

Comment:
This is because the unix port does this as well, right?

Reply:
It's 1 on every port other than a couple of corner cases where it doesn't work. I guess this could be in a different commit, I must have missed it when separating out the other changes.